[ 2025-07-04 15:07:25,147 ] 17 root - ERROR - Error occurred in script: C:\Users\emman\OneDrive\Documents\Projects\flu-shot-learning\push_data.py at line number: 44 with message: connection closed
